force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -163,9 +189,10 @@ public with sharing class ClassesObjectsMethods {
      */
     public static Integer generateRandomNumber() {
         // Use Math class to generate number between 50 and 100
+        Integer rando = (Integer)math.floor(Math.random() * (100 - 50 + 1) + 50);
```

Comment:
The math.floor isn't necessary here. you can just cast Math.random() * (100 - 50 + 1) + 50 to an Integer. Or even Math.random() * (51) + 50